docs/quicktips/update-manager.md

```diff
@@ -13,7 +13,7 @@ description: >-
 
 ## Moonraker.conf
 
-Add the following section to your printers moonraker.conf:
+To enable updates in Mmainsail's web interface, add the following section to your printer's `moonraker.conf`:
```

Comment:
Mainsail :-)

Reply:
Fixed in https://github.com/mainsail-crew/mainsail/pull/518/commits/f2136cb3356f260818df7417805da636b840d14f - thanks!